libkbfs/key_manager.go

```diff
@@ -653,6 +710,20 @@ func (km *KeyManagerStandard) Rekey(ctx context.Context, md *RootMetadata, promp
 	km.config.Reporter().Notify(ctx, rekeyNotification(ctx, km.config, resolvedHandle,
 		false))
 
+	// Save the previous TLF crypt key. It's symmetrically encrypted and appened to a list
+	// for MDv3 metadata.
+	var prevTlfCryptKey TLFCryptKey
+	if currKeyGen >= KeyGen(FirstValidKeyGen) && !md.AreHistoricTLFCryptKeysPerDeviceEncrypted() {
```

Comment:
Meh, an issue with this is after calling `RootMetadata.NewKeyGeneration` the per-device bundles are wiped out which is why I was saving it before that point.